src/Signup.js

```diff
@@ -27,37 +27,37 @@ export default function Signup(props) {
     const { name, value } = event.target;
     if (name === "password")
       setPerson({
-        firstname: person["firstname"],
-        lastname: person["lastname"],
+        firstName: person["firstName"],
+        lastName: person["lastName"],
         email: person["email"],
         password: value,
       });
     else if (name === "email")
       setPerson({
-        firstname: person["firstname"],
-        lastname: person["lastname"],
+        firstName: person["firstName"],
+        lastName: person["lastName"],
         email: value,
         password: person["password"],
       });
-    else if (name === "lastname")
+    else if (name === "lastName")
       setPerson({
-        firstname: person["firstname"],
-        lastname: value,
+        firstName: person["firstName"],
+        lastName: value,
         email: person["email"],
         password: person["password"],
       });
     else
       setPerson({
-        firstname: value,
-        lastname: person["lastname"],
+        firstName: value,
+        lastName: person["lastName"],
         email: person["email"],
         password: person["password"],
       });
   }
 
   function submitForm() {
     console.log("Submit Form Person: " + person);
-    //props.handleSubmit(person);
+    props.handleSubmit(person);
     setPerson({ firstname: "", lastname: "", email: "", password: "" });
     navigate("/home");
```

Comment:
After the user signs up, you should set the credentials according and not to all blank.